src/styles/mixins.scss

```diff
@@ -0,0 +1,31 @@
+@mixin on-tablet {
+    @media (min-width: 576px) {
+        @content;
+    }
+}
+
+@mixin on-desktop {
+    @media (min-width: 1024px) {
+        @content;
+    }
+}
+
+@mixin page-grid {
+    --columns: 2;
```

Comment:
usually we declare SCSS variables using $ instead of --. Then you can call it without using var() notation, just to simplify things a little bit

Reply:
I forgot to check that 👀

Reply:
Also, my demo link is linking to my read.me idk why...